packages/sdk/package.json

```
@@ -27,11 +27,13 @@
   "bugs": {
     "url": "https://github.com/KeystoneHQ/keystone-airgaped-base/issues"
   },
+  "peerDependencies": {
+    "react": "*",
+    "react-dom": "*",
```

```suggestion
    "react-dom": "*"
```

no trailing comma in json @jordansexton